modules/ROOT/pages/manage-message-flows.adoc

```diff
@@ -0,0 +1,206 @@
+= Manage Message Flows
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+After your message flow is running and you are using it with your partners, you should not have to change it much.
+There are certain cases where you might need to make changes, for example:
+
+* You are no longer trading with a particular partner and need to delete the message flow you created for them.
+
+* You completed the testing for a deployed message flow in a pre-production environment, and now want to reclaim those previously allocated Mule runtime resources.
+
+After a message flow is deployed, you can test it, undeploy it, and delete it.
+
+== Test Message Flows
+
+You can test your message flow after you deploy it to ensure that you are receiving messages and any acknowledgments from your partners.
+You should ask your partner to confirm receipt of the transmission once you deploy it to your production environment.
+You can also test the flow by deploying it in a sandbox environment, then undeploying it, and then deploying it to a production environment.
+
+Test your flow in your production environment:
+
+. Drop an EDI file on the receiving endpoint.
+
+. From within your Anypoint Partner Manager environment (either Sandbox or Production) select *Activity* and check for the transmission.
+
+. Check the transmission details for the flow, specifically:
+
+** Check for any errors, and review any that you see for the partner.
+
+** Review the received payload by clicking the transmission and opening the payload info.
+
+** Review and confirm that any TA1 and functional acknowledgments are there.
+
++
+
+. Ensure that the messages are routed to the correct message flow from the target system. For example:
+
+** It was successfully delivered to the target endpoint.
+
+** The message was transformed correctly.
+
+== Edit Message Flows
+
+(Reviewers, I don't have this section fully fleshed out yet.)
+
+You can make changes to a message flow such as adding receiving endpoints and certificates;
+modifying your DataWeave maps; plus changing the associated message types and target endpoints.
+
+Overview of steps include:
+
+* Change receiving endpoints, (non AS2 Identifiers???), and certificates.
+
+* Change associated message types (X12 or JSON) and acknowledgments.
+
+* Upload different DataWeave maps.
+
+* Change associated target message types (X12 or JSON) and endpoints.
+
+You handle this by opening an existing message flow and make your changes directly in the sections you intend to modify.
+
+Consider that any changes to a deployed message flow may impact future message flows. (Reviewers, I'm trying to alert the users to any issues that can come up from changing a deployed message flow.)
```

Comment:
I'm not sure we should include this warning. Maybe we could say something like "modifying identifiers will impact deployed message flows" or similar

Reply:
Fixed.